Astro 7/11 release notes (#2548)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 11, 2023

### Send Astro alerts to email

You can now send Astro alerts to multiple email addresses. Sending Astro alerts to email requires no configuration outside of Astro, which makes it a quick option to improve your alerting infrastructure. See [Configure Astro alerts](https://docs.astronomer.io/astro/alerts?tab=Email#step-1-configure-your-communication-channel) for setup steps.

### Configure SCIM provisioning for Azure AD

If your Organization uses Azure for single sign-on (SSO), you can now set up SCIM provisioning for Astro. SCIM provisioning simplifies user management by allowing you to add and remove Astro users from Okta based on your existing user groups. See [Set up SCIM provisioning](https://docs.astronomer.io/astro/set-up-scim-provisioning?tab=Azure#setup) for more information.

### Additional improvements

> - The `astronomer_monitoring_dag` has been removed from all Deployments on Astro Hosted.

Reviewer: ```suggestion
- The `astronomer_monitoring_dag` has been paused for all image-based Deployments on Astro Hosted, and removed from all DAG-based Deployments. Previously this DAG was used to monitor the deployment's health; this is now replaced with a more robust alternative while allowing the workers to be scaled to 0 when not used.
```

> - The `astronomer_monitoring_dag` has been removed from all Deployments on Astro Hosted.

Reviewer 2: I think:
> - On Astro Hosted deployments, the `astronomer_monitoring_dag` has been paused for all image-based deployments and removed entirely from all deployments with DAG deploys enabled. Previously this DAG was used to monitor the deployment's health; this is now replaced with a more robust alternative while allowing the workers to be scaled to 0 when not used.

> - The `astronomer_monitoring_dag` has been removed from all Deployments on Astro Hosted.

Reviewer 3: (based on Ian's and Felix's feedback)

> - On Astro Hosted deployments, the [`astronomer_monitoring_dag`](runtime-image-architecture.md#astro-monitoring-dag-hybrid-only) has been paused for all image-based Deployments and removed entirely from all Deployments with DAG deploys enabled. It has been replaced with an implementation that allows workers on Deployments to fully scale to 0.

Author: ```suggestion
- On Astro Hosted deployments, the [`astronomer_monitoring_dag`](https://docs.astronomer.io/astro/runtime-image-architecture#astro-monitoring-dag-hybrid-only) has been paused for all image-based Deployments and removed entirely from all Deployments with DAG deploys enabled. It has been replaced with an implementation that allows workers on Deployments to fully scale to 0.
```

> - On Astro Hosted deployments, the `astronomer_monitoring_dag` has been paused for all image-based deployments and removed entirely from all deployments with DAG deploys enabled. Previously this DAG was used to monitor the deployment's health; this is now replaced with a more robust alternative while allowing the workers to be scaled to 0 when not used.

Author: ```suggestion
- On Astro Hosted deployments, the [`astronomer_monitoring_dag`](runtime-image-architecture.md#astro-monitoring-dag-hybrid-only) has been paused for all image-based Deployments and removed entirely from all Deployments with DAG deploys enabled. It has been replaced with a more robust alternative that allows workers on Deployments to fully scale to 0.
```
